OpenXmlPowerTools/HtmlToWml/OpenXMLDocumentPartWrapper.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Drawing.Imaging;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Xml.Linq;
+using DocumentFormat.OpenXml.Packaging;
+using OpenXmlPowerTools;
+using OpenXmlPowerTools.HtmlToWml;
+using OpenXmlPowerTools.HtmlToWml.CSS;
+
+namespace OpenXmlPowerTools.HtmlToWml
+{
+    /// <summary>
+    /// common interface to access the document parts common properties for the MainDocumentPart HeaderPart and FooterPart - they don't seem to have a common interface
+    /// </summary>
+    internal interface IOpenXMLDocumentPartWrapper
+    {
+        IEnumerable<ImagePart> ImageParts { get; }
+        ImagePart AddImagePart(ImagePartType partType, string id);
+        HyperlinkRelationship AddHyperlinkRelationship(Uri hyperlinkUri, bool isExternal, string id);
+    }
+
+    internal class OpenXMLDocumentPartWrapper : IOpenXMLDocumentPartWrapper
+    {
+        private dynamic documentPart;
```

Comment:
Do you need to use dynamic? You'll have to add a reference to `Microsoft.CSharp` if you do. However, I'd rather find a way to not require dynamic - it has quite a bit of overhead. If you really feel it is needed, please document why in the code.

Reply:
I was able to do a workaround didn't want to add the extra library